instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -37,7 +37,9 @@
 
 major_version = elasticsearch.VERSION[0]
```

Comment:
Just curious, [here](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1810/files#diff-ded129d963536138c625950c76c22f4879af17beafb49943c796af9848eea72bR103) you get the version from another attribute.

Reply:
Ah! Changed the src code check in [775bf2b](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1810/commits/775bf2b22e567c2dbca7ec60f600dbd955e86253) to match the existing test.